### ISSUE TITLE
Idea: Add Storybook.ja

### ISSUE THREAD

**Issue by JosephShepin** [stevenson-space/shs#145]
I’ve been eyeing storybook for some time, would like to see if I can build our component library using storybook. 

[storybook](https://storybook.js.org/docs/vue/get-started/introduction)

**Comment by aw-0** [comment]
I like this, would be nice to centralize everything we have & be able to view it all seamlessly.

**Comment by JosephShepin** [comment]
yeah agreed. after doing an initial dive into it, it looks like there are some issues with our SASS and webpack configuration so some significant work will be involved in making it work with our components.

**Comment by aw-0** [comment]
@JosephShepin not sure if you looked any further into this - but would Vite make this easier to accomplish?

**Comment by JosephShepin** [comment]
i think so yeah. i'll take another stab at getting the storybook working
